apisix/plugins/proxy-rewrite.lua

```diff
@@ -61,12 +61,15 @@ local schema = {
             type        = "string",
             enum        = schema_method_enum
         },
+        none_match_abort = {
```

Comment:
It seems that the `none_match_abort` feature has nothing to do with this pr?

Reply:
> It seems that the `none_match_abort` feature has nothing to do with this pr?

please check https://github.com/apache/apisix/pull/9194#issuecomment-1508145751

Reply:
I don't think uri blocking should be done in this plugin, it can be combined with other plugins, keep this plugin simple

Reply:
> I don't think uri blocking should be done in this plugin, it can be combined with other plugins, keep this plugin simple

OK. I'll make another commit to remove this field later.